policyengine_us/variables/gov/states/id/tax/income/credits/grocery/eligible/id_grocery_credit_prorated_eligiblity_fraction.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit_prorated_eligiblity_fraction(Variable):
```

Comment:
To match the verbiage in the tax form
```suggestion
class id_grocery_credit_qualified_months(Variable):
```